spec/player_spec.rb

```diff
@@ -0,0 +1,26 @@
+require './lib/player'
+
+describe Player do
+  before :each do
+    @player_1 = Computer.new
+    @player_2 = Player.new
+  end
+
+  it "has computer player that exists" do
+    expect(@player_2).to be_instance_of(Player)
+  end
+
+  it "has a board to play on" do
+    expect(@player_2.board).to be_instance_of(Board)
+  end
+
+  it "has a differnt board then computer player" do
+    expect(@player_1.board).to be_instance_of(Board)
+    expect(@player_2.board == @player_1.board).to be(false)
```

Comment:
I really like the board not being the same.. because irl duh we have two separate board but having to tell the computer that is interesting to me

Reply:
Thanks! I had to think that through too and go how do we make sure that we are not tieng to put 4 ships on the same board and end up with all kinds of placement errors.